drivers/flash/spi_nor.h

```diff
@@ -24,6 +24,7 @@
 #define SPI_NOR_CMD_PP          0x02    /* Page program */
 #define SPI_NOR_CMD_SE          0x20    /* Sector erase */
 #define SPI_NOR_CMD_BE_32K      0x52    /* Block erase 32KB */
+#define SPI_NOR_CMD_RSFDP       0x5A    /* Read SFDP */
```

Comment:
Could you use the existing `JESD216_CMD_READ_SFDP` instead of adding this here?

Reply:
Sure I will